tests/app/views/test_marketplace.py

```diff
@@ -365,8 +366,10 @@ def test_catalogue_of_briefs_page(self):
                                                                                    self.briefs['meta']['total']
 
     def test_catalogue_of_briefs_page_filtered(self):
-        original_url = "/marketplace/digital-service-professionals/opportunities?" \
-            "page=2&status=live&lot=lot-one&lot=lot-three"
+        with self.app.app_context():
+            original_url = url_for('main.list_opportunities', framework_slug='digital-service-professionals') + \
+                "?page=2&status=live&lot=lot-one&lot=lot-three"
```

Comment:
You can put all these URL parameters into url_for.  Extra parameters get put after the "?".  I.e.:

url_for('main.list_opportunities', framework_slug='digital-service-professionals', page=2, status='live', lot=['lot-one','lot-three'])

should work.